# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1390657211 -3600
#      Sat Jan 25 14:40:11 2014 +0100
# Branch stable
# Node ID 4da51757372f51c1f725b271493057ab784dba3b
# Parent  d2aa3948cec87a8c9495989448369a05781799e1
*de_DE.ts: consistent translation of the two Find Files in the file browser

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -494,17 +494,17 @@ wurde gelöscht oder umbenannt. Soll die Datei jetzt gespeichert werden?%2</translation>
     <message>
         <location line="+17"/>
         <source>Actions on current directory</source>
         <translation>Aktionen mit aktuellem Verzeichnis</translation>
     </message>
     <message>
         <location line="+373"/>
         <source>Find Files ...</source>
-        <translation>Dateien suchen ...</translation>
+        <translation>Dateien suchen...</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>New File</source>
         <translation>Neue Datei</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -545,17 +545,17 @@ wurde gelöscht oder umbenannt. Soll die Datei jetzt gespeichert werden?%2</translation>
         <location line="+12"/>
         <source>Search Directory...</source>
         <translation>Verzeichnis suchen...</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Find Files...</source>
         <translatorcomment>Dateien suchen...</translatorcomment>
-        <translation>Suche Dateien...</translation>
+        <translation>Dateien suchen...</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>New File...</source>
         <translation>Neue Datei...</translation>
     </message>
     <message>
         <location line="+3"/>
